### DISCUSSION TITLE
Reading tiff tag "XPosition" and "YPosition"

### DISCUSSION THREAD

**Discussion by ajay1685** [libvips/libvips#3581] [Q&A] [answered]
Is there a way to read ```XPosition``` and ```YPosition``` from the tiff tag ? I have tried ```xoffset``` and ```yoffset``` but it returns  ```0``` for me. I can see the tiff tag values for ```XPosition``` and ```YPosition``` using ```tiffinfo``` command from libtiff.

**Comment by jcupitt** [comment, marked as answer]
Hi @ajay1685, no, libvips doesn't read those tags.
